### PR TITLE
fix: cleanup fields

### PR DESCRIPTION
**Pull request by shanejonas**
image

image
image

### CODE REVIEW

File: src/components/BlockCard/BlockCard.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ interface IProps {
 
 export default function BlockCard(props: IProps) {
   const { block } = props;
-  const d = moment(new Date(parseInt(block.timestamp, 16) * 1000).toISOString()).format('MMMM Do YYYY, h:mm:ss a')
+  const d = moment(new Date(parseInt(block.timestamp, 16) * 1000).toISOString()).format("MMMM Do YYYY, h:mm:ss a");
```
Comment (zcstarr) [inline src/components/BlockCard/BlockCard.tsx]: what is the Do vs DD ?

File: src/components/BlockView/BlockView.tsx [hunk 2]
```diff
@@ -16,6 +17,9 @@ function BlockView(props: any) {
     timestamp, hash, parentHash, miner, nonce, difficulty,
     extraData, stateRoot, transactionsRoot, receiptsRoot, transactions,
   } = block;
+  const d = moment(
+    new Date(parseInt(timestamp, 16) * 1000).toISOString(),
```
Comment (zcstarr) [inline src/components/BlockView/BlockView.tsx]: should this be a utility function we call create date object from timestamp in muiltiple places with the same formatting.

File: src/components/BlockCard/BlockCard.tsx (hunk 1 above)
Comment (shanejonas) [reply src/components/BlockCard/BlockCard.tsx]: i think it adds the postfix `th`

image

File: src/components/BlockCard/BlockCard.tsx [hunk 3]
```diff
@@ -24,7 +23,8 @@ export default function BlockCard(props: IProps) {
         </CardHeader>
         <CardContent>
           <Typography variant="caption" style={{ fontSize: "11px" }}>{block.hash}</Typography>
-          <Typography gutterBottom>{d}</Typography>
+          <Typography gutterBottom>{hexToDate(block.timestamp)}</Typography>
+          <Typography gutterBottom>{new Buffer(block.extraData.substring(2), "hex").toString()}</Typography>
```
Comment (shanejonas) [inline src/components/BlockCard/BlockCard.tsx]: this could be another helper function `hexToString`

Comment (zcstarr) [reply src/components/BlockCard/BlockCard.tsx]: agreed if you're doing that conversion quite a bit. doesn't have to be in this PR up 2 you!

File: src/helpers/hexToString.ts [hunk 4]
```diff
@@ -0,0 +1,5 @@
+const hexToString = (hex: string) => {
+  return new Buffer(hex.substring(2), "hex").toString();
```
Comment (zcstarr) [inline src/helpers/hexToString.ts]: should we check for 0x here?

**Pull request merged**